metadata/site_import/sites/yourSiteId/payment-methods.xml

```diff
@@ -12,4 +12,11 @@
         <enabled-flag>true</enabled-flag>
         <processor-id>ADYEN_CREDIT</processor-id>
     </payment-method>
+      <payment-method method-id="AdyenPOS">
+        <name xml:lang="x-default">AdyenPOS</name>
+        <description xml:lang="x-default">Adyen POS</description>
+        <image>adyen.png</image>
```

Comment:
what kind of logo is this ? would be great if we just have a terminal logo there.